linkml_runtime/utils/schemaview.py

```diff
@@ -592,6 +592,22 @@ def type_parents(self, type_name: TYPE_NAME, imports=True) -> List[TypeDefinitio
         else:
             return []
 
+    @lru_cache()
+    def get_children(self, name: str, mixin: bool = True) -> List[str]:
+        """
+        get the children of an element (any class, slot, enum, type)
+        :param name: name of the parent element
+        :param mixin: include mixins
+        :return: list of child element
+        """
+        children = []
+        for e, el in self.all_elements().items():
+            if "is_a" in el and el["is_a"] == name:
```

Comment:
I prefer not accessing the dict directly

e.g.

```
if isinstance(el, (ClassDefinition, SlotDefinition, EnumDefinition)):
  if el.is_a == name:
...
```

note also TypeDefinition is not totally symmetric, it uses typeof
```

Reply:
fixed.